src/App.vue

```diff
@@ -241,10 +239,42 @@
 </style>
 
 <style lang="scss" scoped>
-	// modal layout
+
+	.modal-mask {
+		background-color: rgba(0, 0, 0, 0.7);
+
+		& /deep/ .modal-wrapper .modal-container {
```

Comment:
better use v-deep alias as /deep/ will not be transpiled and is not supported in all platforms ;)

Reply:
@skjnldsv Do you have a reference for that? vue-loader docs just list /deep/ and >>> https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors but >>> is a scss selector

I couldn't find them in the bundled files, so I assume they are transpiled just fine

Reply:
https://github.com/vuejs/vue-loader/issues/913

Reply:
Issue is that /deep/ was not transpiled as this was a soon to be added css property. So it stayed this way. But it's now removed

Reply:
https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors

Reply:
Ah thanks. :+1: